floyd/cli/experiment.py

```diff
@@ -164,26 +165,27 @@ def delete(ids, yes):
     """
     Delete project runs
     """
-    
+    failures = False
     for id in ids:
         experiment = ExperimentClient().get(id)
-        task_instance_id = get_module_task_instance_id(experiment.task_instances)
-        task_instance = TaskInstanceClient().get(task_instance_id) if task_instance_id else None
-
+        if not experiment:
+            failures = True
+            continue
 
-        if experiment.state in ["queued", "running"]:
```

Comment:
Can we keep this check? Without this the cli will delete the module even when the experiment is running which is not ideal.